mrc/control/controller.py

```diff
@@ -0,0 +1,20 @@
+from threading import Thread
+from typing import Callable
+
+from mrc.localization.locator import Locator
+
+
+class Controller(Thread):
+    def __init__(self, locator: Locator, dtp_func: Callable([float, float], None)):
+        super().__init__()
+        self.dtp_func = dtp_func
+        self.locator = locator
+        self.running = False
+
+    def run(self):
+        self.running = True
+        while True:
```

Comment:
Why not while self.running? And"stop" method setting self.running to False could be useful.

Reply:
Done